sashimi-webapp/src/logic/documentPackager/markdownProcessor.js

```diff
@@ -21,6 +21,7 @@ import mdAsciiMath from './markdown-it-asciimath';
 import mdDiagrams from './markdown-it-diagram';
 // Custom conditional plugin
 import mdConditional from './conditionalProcessor';
+import mdLineNumber from './plugins/markdown-it-line-number';
```

Comment:
@amoshydra  Should we refactor and import all self-written/customised plugins from plugins folder?